test/unit_tests/test_translation.py

```diff
@@ -297,6 +366,47 @@ def test_translate_operation_with_unique_params(
         )
 
 
+def test_translate_operation_parametrized_evolution():
+    """Test that a ParametrizedEvolution is translated to a PulseGate correctly."""
+
+    def amplitude(p, t):
+        return p * (np.sin(t) + 1)
+
+    H = transmon_drive(0.02, np.pi, 0.5, [0, 1])
+    H += transmon_drive(amplitude, -np.pi / 2, 0.75, [2, 3])
+
+    amplitude_param = 0.1
+    op = ParametrizedEvolution(H, [amplitude_param], t=50)
+
+    dev = _aws_device(wires=4, device_arn="arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy")
+    braket_gate = translate_operation(op, device=dev)
+
+    assert isinstance(braket_gate, gates.PulseGate)
+    assert braket_gate.qubit_count == 4
+
+    ps = braket_gate.pulse_sequence
+    expected_frames = set(dev._device.frames[f"q{w}_drive"] for w in range(4))
+
+    assert set(ps.frames.values()) == expected_frames
+
+    max_amplitude = dev._device.properties.pulse.validationParameters["MAX_AMPLITUDE"]
```

Comment:
This raises `AttributeError: 'AwsDevice' object has no attribute '_properties'` because the mocked device does not have this property yet